scripts/createBuildScript.ts

```diff
@@ -73,10 +101,19 @@ ${Object.entries(shortcuts)
     shift
 done`,
         'total_start_time=$(date +%s%N)',
+        "NODE_VERSION=$(grep '\"node\":' ./package.json | awk -F: '{ print $2 }' | sed 's/[\",]//g' | sed 's/\\^v//g' | tr -d '[:space:]')\n" +
```

Comment:
Bash escaped in .ts is pita to read

Reply:
any ideas on how to improve that?

Reply:
Maybe another shell-script which is called by the typescript-script? :D

Reply:
seems like a little overload, doesn't it? :sweat_smile: